src/common.rkt

```diff
@@ -225,7 +225,7 @@
 
 (define ((inv-comparator test) . args)
   (for/or ([left args] [right (cdr args)])
-    (not (test left right))))
+    ((negate test) left right)))
```

Comment:
No, dude, I mean use `(negate (comparator x))` instead of writing a separate `inv-comparator`

Reply:
Lol. Got it